lib/govuk_navigation_helpers/content_item.rb

```diff
@@ -123,5 +151,15 @@ def get_root_taxon(taxon)
         get_root_taxon(parent_taxons.first)
       end
     end
+
+    def check_link_type(links, type)
```

Comment:
Could we do something like this? 

```rb
links.select do |link|
  link["document_type"] == type
end
```

Reply:
And perhaps rename the method to something like `filter_link_type` to indicate what it's doing.